shared/login/register/code-page/qr/index.ios.js

```diff
@@ -11,6 +11,7 @@ class QR extends Component<void, Props, void> {
       return (
         <Camera
           style={{...cameraStyle, ...this.props.style}}
+          captureAudio={false}
```

Comment:
this should have always been set. in ios 10 you get extra microphone popups w/o this